subprojects/docs/src/docs/userguide/nativeBinaries.xml

```diff
@@ -311,7 +311,7 @@
                 of the "operators" component, you would run the <userinput>assembleDependentsOperatorsPassingStaticLibrary</userinput> task:
             </para>
             <sample id="nativeAssembleDependentComponents" dir="native-binaries/cunit" title="Assemble components that depends on the passing/static binary of the operators component">
-                <output args="assembleDependentsOperatorsPassingStaticLibrary"/>
+                <output args="assembleDependentsOperatorsPassingStaticLibrary --max-workers=1"/>
```

Comment:
I'm not sure I fully understand why we need `--max-workers=1` here. Could you write a small note a to why?